app/services/datetime_parser.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class DatetimeParser
+  def initialize(date:, time:)
+    @raw_date = date.to_h
+    @raw_time = time.to_s
+  end
+
+  def parse
+    @issue_items = []
+
+    check_date_is_valid
+    check_time_is_valid
+    return if issue_items.any?
+
+    Time.current.change(
```

Comment:
I think you can just use `Time.new` here as no specific part of the date/time is being changed.

Reply:
Hmmm... with `Time.current.change`, all tests work as before. But if I change to `Time.new`, then the datetime is saved slightly differently and the `to_s` is formatted differently ("2030-04-20 10:26:00 +0000" instead of "2030-04-20 10:26:00 UTC"), causing an expectation to fail in managing_step_by_step_pages_spec.rb.

Presumably this is because `Time.new` doesn't contain any timezone information, whereas `Time.current` does. I'm inclined to keep it as `Time.current.change` as per Content Publisher, so that it's clear going from one input to separate date/time inputs doesn't change how the value is stored. What do you think?

Reply:
Can do `Time.new(...).utc` 👍